src/backend/api/Fusion.Resources.Api/Notifications/RequestsNotificationHandler.cs

```diff
@@ -41,10 +41,14 @@ public async Task Handle(RequestDeclinedByCompany notification, CancellationToke
 
             foreach (var recipient in recipients)
             {
-                await notificationClient.CreateNotificationAsync(n => n
```

Comment:
Fusion.Integration.Notification 5.1.0 required change.
Commented code can be removed.

Reply:
Remove commented code